nvme_dimm_fix.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -uex
+
+tnode=${NODELIST##*,}
+
+idstr='Non-Volatile memory controller: '
+idstr+='Intel Corporation NVMe Datacenter SSD.*Optane'
+# shellcheck disable SC2029
```

Comment:
(lint) Couldn't parse this shellcheck annotation. [SC1073]
(lint) Expected "=". Fix any mentioned problems and try again. [SC1072]